core/src/main/java/org/jsmart/zerocode/core/httpclient/utils/FileUploadUtils.java

```diff
@@ -57,7 +57,7 @@ public static void buildOtherRequestParams(Map<String, Object> fileFieldNameValu
             if (entry.getKey().equals(FILES_FIELD) || entry.getKey().equals(BOUNDARY_FIELD)) {
                 continue;
             }
-            multipartEntityBuilder.addPart(entry.getKey(), new StringBody((String) entry.getValue(), TEXT_PLAIN));
+            multipartEntityBuilder.addPart(entry.getKey(), new StringBody(entry.getValue().toString(), TEXT_PLAIN));
```

Comment:
Looks good.
Can you add a Unit test for this ?

Reply:
@nirmalchandra Do you need me to add the unit test case for the full file? I cannot add unit test case for that specific part of code because there is no UT class for FileUploadUtils.

Reply:
We need to make sure the new code fixes the issue. How do we ensure that?
>  multipartEntityBuilder.addPart(entry.getKey(), new StringBody(entry.getValue().toString(), TEXT_PLAIN)); 

You can create a new test file called `FileUploadUtilsTest.java` to the `test` package.
First Unit test, followed by an an integration test if you can. 
Otheriwse we are not sure aout this code might break existing functionalities.

Reply:
@nirmalchandra New test class & test cases added to test the method updated in the fix. Check FileUploadUtilsTest.java.